pandas/tests/indexes/multi/test_get_level_values.py

```diff
@@ -0,0 +1,13 @@
+from pandas import MultiIndex, Timestamp, date_range
+
+
+class TestGetLevelValues:
```

Comment:
This method is also on Index. should the module be just pandas/tests/indexes/test_get_level_values.py

Reply:
sorry... pandas/tests/indexes/methods/test_get_level_values.py

Reply:
it exists on Index, but is really only relevant for MultiIndex i think

Reply:
I appreciate that, but I assume that this reorganization is to aid parametrisation. and that the index tests will be grouped by method instead of Index class?

Reply:
> the index tests will be grouped by method instead of Index class

I have no plans to do that anytime soon

Reply:
fair enough, I suppose that Series and Frame are still separate and that's only two classes. we'll see how the method based testing and parametrisation goes on those.